service-base/src/main/java/org/eclipse/hono/service/credentials/CredentialsHttpEndpoint.java

```diff
@@ -37,6 +39,7 @@
  * It receives HTTP requests representing operation invocations and sends them to the address {@link CredentialsConstants#CREDENTIALS_ENDPOINT} on the vertx
  * event bus for processing. The outcome is then returned to the client in the HTTP response.
  */
+@Deprecated
```

Comment:
I think adding a `@deprecated` JavaDoc indicating what to use instead would be helpful